Mention syntax highlighting in the guidelines

xfq opened the pull request:
Mention syntax highlighting in the editorial guidelines.

Preview:

<img width="711" alt="1" src="https://user-images.githubusercontent.com/2863444/156736828-8d272832-44d7-4614-bf0f-9e9b5d01b375.png">

<img width="1036" alt="2" src="https://user-images.githubusercontent.com/2863444/156736843-56c611bf-0fad-4e57-8cda-e1995c2c3ad1.png">

r12a commented:
Articles containing syntax highlighting will generally be published on /International, so the links should be relative, rather than assuming i18n-drafts exists.

It may be worth pointing people to https://r12a.github.io/app-conversion/ as a way of quickly converting `>` to `&gt;`, etc.

xfq commented:
Thanks, @r12a. I updated the PR:

* changed the links to relative links
* added the guidelines for ReSpec
* added a link to the [Unicode code converter](https://r12a.github.io/app-conversion/)

I also updated the preview.
